mailpoet/lib/Automation/Integrations/WooCommerce/Fields/TermOptionsBuilder.php

```diff
@@ -0,0 +1,58 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\Automation\Integrations\WooCommerce\Fields;
+
+use MailPoet\Automation\Engine\WordPress;
+use WP_Error;
+use WP_Term;
+
+class TermOptionsBuilder {
+  /** @var WordPress */
+  private $wordPress;
+
+  public function __construct(
+    WordPress $wordPress
+  ) {
+    $this->wordPress = $wordPress;
+  }
+
+  /** @return array<array{id: int, name: string}> */
+  public function getCategoryOptions(): array {
```

Comment:
I would rename it to getProductCategoryOptions I think. When reading the function somewhere where it's used it would be immediately clear what you get imo

Reply:
https://github.com/mailpoet/mailpoet/pull/4984/commits/9bebd41eaea85282facc243f5bff6133e24db1a3